### DISCUSSION TITLE
Custom module for Spotify flatpak (as well as the default "custom/media") not showing anything when playing media

### DISCUSSION THREAD

**Discussion by mattkel** [Alexays/Waybar#1822] [Q&A]
Fedora 32 on a Macbook Pro.

Quick Question.

Using the "custom/media" module with the mediaplayer.py from the default config doesn't show anything when playing a song from Spotify (flatpak version) or even a local song/movie using VLC.

I've even tried using other users dotfiles for a "custom/spotify" module which uses a shell script instead of the default python one, yet to no avail.

Could someone who has a custom module working with the flatpak version of spotify please link your dotfiles and/or explain why it might not be working for me? Perhaps a pathing or naming issue? Been stuck on this for a while lol.

Thanks, Matt

**Comment by b-zee** [comment]
Is the module by chance a symbolic link? I've just come across the issue where my custom module is a symbolic link to a (python) script.

**Comment by der-eismann** [comment]
Had the same problem, after removing the `"interval": 30,` line from my custom config it finally worked.

**Comment by hughwilliams94** [comment]
I've also noticed that the script doesn't execute when it is a symlink
